Makefile

```diff
@@ -24,5 +24,8 @@ check: test
 	git status --porcelain
 	/bin/bash -c 'test $$(git status --porcelain | wc -l) == 0'
 test:
+	go get -t ./...
+	go get github.com/golang/lint/golint
 	go test -race ./...
 	go vet ./...
+	golint ./...
```

Comment:
I would love to have a target that just runs:
```
go test -race ./...
go vet ./...
golint ./...
```
Actually, all I care about is `go test -race ./...` I need this target because it is what I run from terminal during debugging... I can't remember to spell `go test -race ./...` every time.. :)

We could totally call it `make runtest` or something... or `make go-test`, I moved the git checks into `make check` for this reason...